Libraries/LibWeb/HTML/NavigateEvent.cpp

```diff
@@ -25,6 +25,13 @@ namespace Web::HTML {
 
 GC_DEFINE_ALLOCATOR(NavigateEvent);
 
+GC::Ref<NavigateEvent> NavigateEvent::create(JS::Realm& realm, FlyString const& event_name, NavigateEventInit const& event_init)
+{
+    auto event = realm.create<NavigateEvent>(realm, event_name, event_init);
+    event->set_is_trusted(true);
```

Comment:
Can we import the test?

Reply:
unfortunately not, our harness cant handle it, pressumably because it is file scheme navigation to query instead of http